app/javascript/application.js

```diff
@@ -3,8 +3,12 @@
 // https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#javascript
 import { initAll } from 'govuk-frontend'
 import accessibleAutocomplete from 'accessible-autocomplete'
+import '@hotwired/turbo-rails'
+
 initAll()
 
+Turbo.setFormMode('optin')
```

Comment:
Question: does this mean we can remove the `data: { turbo: 'false' }` stuff we have been adding everywhere?

Reply:
Thats my understanding!